### PR TITLE
fix: fixed UI issues of discussion for incontext sidebar

### PR DESCRIPTION
**Pull request by sundasnoreen12**
[INF-1184](https://2u-internal.atlassian.net/browse/INF-1184)

### Description

fixed UI issues of discussion for incontext sidebar based on Figma [Legacy Notifications vs Discussions Sidebar](https://www.figma.com/proto/PwKdrc6pQhlw5XJaB7hBrb/Legacy-Notifications-vs-Discussions-Sidebar?page-id=0%3A1&type=design&node-id=28-14107&viewport=625%2C3258%2C0.21&t=Cq10PY4QgOnlrpuh-9&scaling=scale-down&starting-point-node-id=28%3A14107&hotspot-hints=0&show-proto-sidebar=1).

#### Screenshots/sandbox (optional):
Screenshot 2023-12-27 at 2 22 42 PM

Screenshot 2023-12-27 at 2 20 37 PM

#### Merge Checklist

* [ ] If your update includes visual changes, have they been reviewed by a designer? Send them a link to the Sandbox, if applicable.
* [ ] Is there adequate test coverage for your changes?

#### Post-merge Checklist

* [ ] Deploy the changes to prod after verifying on stage or ask **@openedx/edx-infinity** to do it. 
* [ ] 🎉 🙌 Celebrate! Thanks for your contribution.

### CODE REVIEW

File: src/discussions/posts/post-actions-bar/PostActionsBar.jsx [hunk 1]
```diff
@@ -67,15 +67,16 @@ const PostActionsBar = () => {
...
+          <div className="border-right border-light-300 mr-2 my-10px" />
+          <div className="justify-content-center my-2">
             <IconButton
               src={Close}
               iconAs={Icon}
               onClick={handleCloseInContext}
               alt={intl.formatMessage(messages.close)}
               iconClassNames="spinner-dimensions"
-              className="spinner-dimensions"
+              className="spinner-dimensions-36 text-black"
```
Comment (awais-ansari) [inline src/discussions/posts/post-actions-bar/PostActionsBar.jsx:78]: use `text-primary-500` instead of `text-black`

File: src/index.scss [hunk 2]
```diff
@@ -471,6 +466,11 @@ header {
   width: 24px !important;
 }
 
+.spinner-dimensions-36 {
+  height: 36px !important;
+  width: 36px !important;
+}
+
```
Comment (awais-ansari) [inline src/index.scss:473]: No need to change the spinner dimensions.

File: src/discussions/posts/post-actions-bar/PostActionsBar.jsx [hunk 3]
```diff
@@ -67,15 +67,16 @@ const PostActionsBar = () => {
...
+              variant="light"
```
Comment (awais-ansari) [inline src/discussions/posts/post-actions-bar/PostActionsBar.jsx:79]: is there any reason to add a variant = "light"?

**Review by awais-ansari — CHANGES_REQUESTED** [review]

File: src/index.scss (hunk 2 above)
Comment (sundasnoreen12) [reply src/index.scss:473]: The "spinner-dimensions" class was previously utilized with dimensions specified as 24px for both height and width, whereas the Figma design indicates dimensions of 36px.

Comment (awais-ansari) [reply src/index.scss:473]: We are using a spinner- dimensions 24px all over the app. Please revert this change to 24px to make it consistent.

File: src/discussions/posts/post-actions-bar/PostActionsBar.jsx [hunk 4]
```diff
@@ -67,15 +67,15 @@ const PostActionsBar = () => {
...
+          <div className="border-right border-light-300 mr-2 my-10px" />
+          <div className="justify-content-center my-2">
             <IconButton
               src={Close}
               iconAs={Icon}
               onClick={handleCloseInContext}
               alt={intl.formatMessage(messages.close)}
               iconClassNames="spinner-dimensions"
-              className="spinner-dimensions"
+              className="spinner-dimensions-36 text-primary-500"
```
Comment (awais-ansari) [inline src/discussions/posts/post-actions-bar/PostActionsBar.jsx:78]: Do not need to explicitly define `text-primary-500`. IconButton component default variant is primary. The primary variant uses the same color as defined in Figma.

**Review by awais-ansari — CHANGES_REQUESTED** [review]

**Review by awais-ansari — APPROVED** [review]

**Review by ayesha-waris — APPROVED** [review]
PR LGTM

**Pull request merged**